src/engine/engine.cpp

```diff
@@ -434,23 +434,23 @@ const char* DivEngine::getSystemName(DivSystem sys) {
     case DIV_SYSTEM_VRC6:
       return "Konami VRC6";
     case DIV_SYSTEM_OPLL:
-      return "Yamaha OPLL";
+      return "Sega FM Sound Unit";
     case DIV_SYSTEM_FDS:
       return "Famicom Disk System";
     case DIV_SYSTEM_MMC5:
-      return "MMC5";
+      return "Nintendo MMC5";
```

Comment:
Nintendo will take this repo down for using their name...

Reply:
will revert